index.html

```diff
@@ -49,242 +54,83 @@ <h4 class="headline">LET’S CONNECT</h4>
       </ul>
     </div>
   </section>
-  
-  <section id="portfolio">
+
+  <section class="card-section">
     <div class="cards-container">
       <ul class="cards-list flex-center-culomn">
```

Comment:
- [ ] Fantastic work!!! @BouaikIlham, I really love this implementation. Keep it up 👏